TodoApp/src/TodoApp.Contracts/Helpers/IServiceHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace TodoApp.Contracts.Helpers
+{
+    public interface IServiceHelper
```

Comment:
This is a bit worse than the namespace - it is a `>/dev/null` class :rofl: Seriously, the name should tell the reader what is inside a class, what to expect from an instance. You wrap some static implementations here, so maybe choose a name along this line.